a10_octavia/tests/unit/common/test_config_types.py

```diff
@@ -0,0 +1,99 @@
+#    Copyright 2020, A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+
+import json
+
+from oslo_config import cfg
+from oslo_config import fixture as oslo_fixture
+
+from octavia.tests.unit import base
+
+from a10_octavia.common import config_options
+from a10_octavia.common import config_types
+from a10_octavia.common import data_models as data_models
+from a10_octavia.tests.common import a10constants
+
+CONF = cfg.CONF
+
+RACK_DEVICE_1 = {
+    "project_id": "project-1",
+    "ip_address": "10.0.0.1",
+    "device_name": "rack_thunder_1",
+    "username": "abc",
+    "password": "abc",
+    "interface_vlan_map": {"5": {"11": {"use_dhcp": "True"},
+                                 "12": {"ve_ip_address": ".10"}}}
+}
+
+RACK_DEVICE_2 = {
```

Comment:
Just a note : (not part of review comments)

Since we will be using `RACK_DEVICE` dummy config frequently, we can shift this to a common place. Maybe not in this PR, but whoever will use it again can take up this task to refactor this.